ltdconveyor/s3upload.py

```diff
@@ -355,6 +398,10 @@ def list_dirnames_in_directory(self, dirname):
 
         if '.' in dirnames:
             dirnames.remove('.')
+
```

Comment:
If there's ever going to be more than these two, you might want to do this as a loop over a list of to-remove strings, but there probably won't be.